modules/admin_manual/pages/configuration/server/caching_configuration.adoc

```diff
@@ -271,6 +274,20 @@ use this example configuration:
 ],
 ----
 
+For setting up Redis to be accessed via a socket from from webserver user, consider the following settings:
```

Comment:
```suggestion
If setting up Redis to be accessed via a socket from from webserver user, then consider the following settings:
```
(and are these just recommendations to consider, or do you actually have to do these?